analytics/filesystem/file_module.go

```diff
@@ -85,6 +85,12 @@ func (f *FileLogger) LogNotificationEventObject(ne *analytics.NotificationEvent)
 	f.Logger.Flush()
 }
 
+// Shutdown the logger
+func (f *FileLogger) Shutdown() {
+	// clear all pending buffered data in case there is any
```

Comment:
Just to follow the same pattern of the agma module, should we `glog.Infof`?
```
88   // Shutdown the logger
89   func (f *FileLogger) Shutdown() {
90       // clear all pending buffered data in case there is any
   +     glog.Infof("[FileLogger] Shutdown, trying to flush buffer")
91       f.Logger.Flush()
92   }
analytics/filesystem/file_module.go
```

Reply:
there is no additional args to template the log message, so `glog.Info` is sufficient.